client/src/pages/MemberProfilePage/MemberProfilePage.js

```diff
@@ -35,7 +35,11 @@ class MemberProfilePage extends Component {
   }
```

Comment:
Are you using this component? Can you remove these changes from your PR by resetting this file?

Reply:
Can you reset this file so it's not part of this PR?